README.md

```diff
@@ -1,49 +1,69 @@
 # useeio.js
-`useeio.js` is a JavaScript client API for the [USEEIO
-API](https://github.com/USEPA/USEEIO_API) that runs in the browser. It is
-written in TypeScript and uses [rollup.js](https://rollupjs.org) with the
-[typescript2 plugin](https://www.npmjs.com/package/rollup-plugin-typescript2)
-to create a single UMD bundle; [terser](https://terser.org/) is then used
-to create a minified bundle.
 
+The useeio.js repo contains a JavaScript client API for the [USEEIO
+API](https://github.com/USEPA/USEEIO_API) that runs in the browser. It is written in TypeScript and uses [rollup.js](https://rollupjs.org) with the [typescript2 plugin](https://www.npmjs.com/package/rollup-plugin-typescript2) to create a single UMD bundle; [terser](https://terser.org/) is then used to create a minified bundle.
 
-## Usage
+## Impact Reports
```

Comment:
We could list this information as an Example application but not under an Impact Reports heading.